CHANGELOG/CHANGELOG-1.1.md

```diff
@@ -0,0 +1,94 @@
+- Add support for the "Reject" action of Antrea-native policies in the Traceflow observations. ([#2032](https://github.com/antrea-io/antrea/pull/2032), [@gran-vmv])
+- Add support for the "endPort" field in K8s NetworkPolicies. ([#2190](https://github.com/antrea-io/antrea/pull/2190), [@GraysonWu])
+- Add support for [dual-stack Services], which are enabled by default in K8s v1.21, in AntreaProxy. ([#2207](https://github.com/antrea-io/antrea/pull/2207), [@xliuxu])
+- Export flow records about connections denied by NetworkPolicies from the FlowExporter and the FlowAggregator; the records include information about the policy responsible for denying the connection when applicable. ([#2112](https://github.com/antrea-io/antrea/pull/2112), [@zyiou])
+- Add more NetworkPolicy-related information to IPFIX flow records exported by the FlowAggregator (policy type and rule name). ([#2163](https://github.com/antrea-io/antrea/pull/2163), [@heanlan])
+- Add live-traffic Traceflow support to the Antrea [Octant] plugin, which includes support for displaying the captured packet's headers. ([#2124](https://github.com/antrea-io/antrea/pull/2124), [#2182](https://github.com/antrea-io/antrea/pull/2182), [@luolanzone])
+- Add crd.antrea.io/v1alpha3/ClusterGroup API resource which removes the deprecated "ipBlock" field; a [conversion webhook] is added to the Controller to convert from the v1alpha2 version to the v1alpha3 version. ([#2008](https://github.com/antrea-io/antrea/pull/2008), [@Dyanngg])
+- Add support for providing an IP address as the source for live-traffic Traceflow; the source can also be omitted altogether in which case any source can be a match. ([#2068](https://github.com/antrea-io/antrea/pull/2068), [@jianjuns])
+- Add ICMP echo ID and sequence number to the captured packet for live-traffic Traceflow. ([#2162](https://github.com/antrea-io/antrea/pull/2162), [@jianjuns])
+- Add support for dumping OVS groups with the "antctl get of" command. ([#1984](https://github.com/antrea-io/antrea/pull/1984), [@jianjuns])
+- Add new "antrea_agent_deny_connection_count" Prometheus metric to keep track of the number of connections denied because of NetworkPolicies; if too many connections are denied within a short window of time, the metric may undercount. ([#2112](https://github.com/antrea-io/antrea/pull/2112), [@zyiou])
+- Generate and check-in clientset code for ClusterGroupMembers and GroupAssociation, to facilitate consumption of these APIs by third-party software. ([#2130](https://github.com/antrea-io/antrea/pull/2130), [@Dyanngg])
+- Package [whereabouts] CNI plugin into the Antrea Linux container image and install the binary on each Node. ([#2185](https://github.com/antrea-io/antrea/pull/2185), [@arunvelayutham])
```

Comment:
Humm.. This one does not provide any extra functionality. What about moving it to Changed?

Reply:
done